dockers/app/Dockerfile

```diff
@@ -1,27 +1,55 @@
-FROM isard/alpine-pandas:latest
+FROM alpine:3.8 as production
 MAINTAINER isard <info@isard.com>
 
-RUN apk add --no-cache bash yarn py3-libvirt py3-paramiko py3-lxml py3-pexpect py3-openssl py3-bcrypt py3-gevent py3-flask py3-netaddr py3-requests curl openssh-client
+RUN apk add --no-cache \
+	yarn \
+	py3-libvirt \
+	py3-paramiko \
+	py3-lxml \
+	py3-pexpect \
+	py3-openssl \
+	py3-bcrypt \
+	py3-gevent \
+	py3-flask \
+	py3-netaddr \
+	py3-requests \
+	curl \
+	openssh-client \
+	supervisor
+
+RUN apk add --no-cache --virtual .build_deps \
+	build-base \
+	python3-dev \
+	libffi-dev \
+	openssl-dev 
+RUN pip3 install pandas
```

Comment:
It should be `pip3 install --no-cache-dir`

Reply:
Fixed